<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoTipWindowClass</class>
 <widget class="QWidget" name="UserInfoTipWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserInfoTipWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QToolButton" name="toolButtonUserInfo">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/app.png</normaloff>:/resources/images/app.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelUserIDInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelDescription">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelExtraInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonEmail">
       <property name="toolTip">
        <string>Send an email</string>
       </property>
       <property name="whatsThis">
        <string>Send an email</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/email.png</normaloff>:/resources/images/email.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
